index.html

```diff
@@ -197,9 +198,15 @@ <h3>Working Group Notes, registries etc</h3>
         <li><a href="https://github.com/w3c/png-hdr-pq/pulse">PQ HDR in PNG</a></li>
         <li><a href="https://github.com/w3c/tt-profile-registry/pulse">TTML Profile Registry</a></li>
         <li><a href="https://github.com/w3c/ttml-webvtt-mapping/pulse">TTML to WebVTT mapping</a></li>
+        <li><a href="https://github.com/w3c/dapt-reqs">Requirements on Dubbing and Audio description Profiles of TTML2 (DAPT)</a></li>
+      </ul>
+      <h3>Suspended Recommendation Track specification repositories</h3>
+      <ul>
+        <li><a href="https://github.com/w3c/ttml3/pulse">TTML3</a></li>
+        <li><a href="https://github.com/w3c/adpt/pulse">ADPT</a></li>
       </ul>
       <h2 id="join">How to join the group</h2>
-      <p>This group operates under the <a href="https://www.w3.org/2003/12/22-pp-faq.html">W3C
+      <p>This group operates under the <a href="https://www.w3.org/2020/09/15-pp-faq.html">W3C
```

Comment:
Should this not be a link to the Patent Policy itself rather than the FAQ?

```suggestion
      <p>This group operates under the <a href="https://www.w3.org/Consortium/Patent-Policy-20200915/">W3C
```

Reply:
Ah, agree on this. I'm not sure why it was linked to pp-faq, neither. (seems to exist more than five years, per commit history)